packages/sign_in_with_apple/sign_in_with_apple/darwin/Classes/SignInWithAppleError.swift

```diff
@@ -107,15 +107,21 @@ public enum SignInWithAppleError {
                 errorCode = "authorization-error/notHandled"
             case .failed:
                 errorCode = "authorization-error/failed"
-#if (os(iOS) && swift(>=5.5)) || (os(macOS) && swift(>=5.5.1))
+#if compiler(>=5.5)
```

Comment:
Was this change needed? The old comment refers to this being shipped in different versions for iOS and macOS.

But overall I would not mind to change this, as we only support Xcode 14.3.1 (with Swift 5.8), so this will work on either OS with this check and that macOS version.

Just generally of the "don't fix what's not broken" camp here, as we don't know if there is not some user relying on this behavior because of a certain old Xcode version (though by now there might be 3 other changes that would've long prevented that one from using the latest version of this package, as we can not test every single combination).

Reply:
I have reverted the change